### ISSUE TITLE
Marketing API: Forms support

### ISSUE THREAD

**Issue by gmpf** [HubSpot/hubspot-api-php#46]
Hey, I'm basically looking for this client's equivalent of the v2 API client's form retrieval. In the v2 API client, I suppose it would be something like this:
```php
$form = $hubspot->forms()->geById(MY_FORM_ID);
```
This v3 API client implementation does not seem to support forms (yet?), i. e. there is no abstraction here for a v3 API call like this:
```bash
curl --request GET \
  --url 'https://api.hubapi.com/marketing/v3/forms/formId?hapikey=YOUR_HUBSPOT_API_KEY'
```
Is that correct or am I just looking in the wrong place?

**Comment by robertodormepoco** [comment]
+1 Basically the marketing API is completely missing as far as i can see

**Comment by ksvirkou-hubspot** [comment]
Hi 
Form API is currently under development 
as soon as it is ready we will publish it

**Comment by gmpf** [comment]
@ksvirkou-hubspot Thank you for confirming, I'll use the v2 API for now then.

**Comment by hitty** [comment]
Hi, @ksvirkou-hubspot 
I understand correctly that there are no forms yet in API version 3?

**Comment by ksvirkou-hubspot** [comment]
> Hi, @ksvirkou-hubspot I understand correctly that there are no forms yet in API version 3?

You understand correctly 
Still no Form API v3

**Comment by apeisa** [comment]
What is the latest API for form entry submit? This one has v3 and "legacy" both in the url: https://legacydocs.hubspot.com/docs/methods/forms/submit_form_v3_authentication

Is it what one should use if the need is to submit form entries?

**Comment by CBRPLX** [comment]
@apeisa Even if it's not supported yet, I use the v3 client, but with the "no wrapped endpoint" feature as follows :

`$response = $hubspot->apiRequest([
    'method' => 'POST',
    'baseUrl' => 'https://api.hsforms.com',
    'path' => "/submissions/v3/integration/submit/$hubspotPortalId/$hubspotFormGuid",
    'body' => [
        "fields" => $fieldsValues,
        "context" => $contextValues,
    ],
]);`

This way I'm ready when the submit will be available with client.

Hope it helps you

**Comment by ksvirkou-hubspot** [comment]
Hi everyone.
I've just published forms API in version [10.2.0](https://packagist.org/packages/hubspot/api-client#10.2.0).
But this API doesn't contain all necessary methods.
if there is no necessary method in the SDK use `$hubspot->apiRequest()`.
You can check all available SDK's methods in the [docs](https://developers.hubspot.com/docs/api/marketing/forms).
